runtime/doc/news.txt

```diff
@@ -281,6 +281,8 @@ TERMINAL
   'scrollback' are not reflown.
 • The |terminal| now supports OSC 8 escape sequences and will display
   hyperlinks in supporting host terminals.
+• The |terminal| now updates the cursor shape and blink when set by an
+  application.
```

Comment:
should we also mention something like "defers to the actual UI cursor instead of drawing a fake cursor"